lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1025,26 +1040,373 @@ spec = do
         expectResponseCode @IO HTTP.status404 r
         expectErrorMessage (errMsg404NoWallet $ w ^. walletId) r
 
+    it "TRANS_LIST_01 - Can list Incoming and Outgoing transactions" $ \ctx -> do
+        (wSrc, _, _) <- fixtureWalletManyTxs ctx [1]
+        let (feeMin, feeMax) = ctx ^. feeEstimator $ TxDescription
+                { nInputs = 1
+                , nOutputs = 1
+                }
+
+        -- Verify Tx list contains Incoming and Outgoing
+        r <- request @([ApiTransaction t]) ctx (listTxEp wSrc mempty)
+            Default Empty
+        expectResponseCode @IO HTTP.status200 r
+
+        -- Outgoing tx inputs and outpus size
+        let outsOut = getFromResponseList 0 outputs r
+        let insOut = getFromResponseList 0 inputs r
+        length (outsOut :: [AddressAmount t]) `shouldBe` 2
+        length (insOut :: [ApiTxInput t]) `shouldBe` 1
+
+        -- Incoming tx inputs and outpus size
+        let outsIn = getFromResponseList 1 outputs r
+        let insIn = getFromResponseList 1 inputs r
+        length (outsIn :: [AddressAmount t]) `shouldBe` 1000
```

Comment:
Whola. This sounds like something that is very specific to the way the http-bridge is set up. I wouldn't expect fixtures to yield such transactions! 
Generally speaking, we shouldn't make any assertions on the tx structure of the faucet wallets.

Reply:
removed